src/multiStore.js

```diff
@@ -2,27 +2,73 @@
  * Creates a multi-store
  *
  * @param {store[]} stores - The stores to sync.
+ * @param {object} [errorHandlers] - Functions to override the error handling behavior.
+ * @param {function} [errorHandlers.onGetError] - A function to execute if a store errors when getting a value.
+ * @param {function} [errorHandlers.onSetError] - A function to execute if a store errors when setting a value.
+ * @param {function} [errorHandlers.onClearError] - A function to execute if a store errors when clearing all values.
+ * @param {function} [errorHandlers.onRemoveError] - A function to execute if a store errors when removing a value.
  *
  * @returns {object}
  */
-export default stores => {
+export default (
+  stores, 
+  { onGetError, onSetError, onClearError, onRemoveError } = {}
+) => {
+  const formatError = (err, storeIndex) => {
+    err.currentStore = stores[storeIndex]
+    err.nextStore = stores[storeIndex + 1]
```

Comment:
ESLint will probably dislike that you're mutating a parameter.

Reply:
I don't get any errors or warnings about this when lint 🤷‍♂ Should I wrap it in a custom error you think, exposing the original error in `err.originalError`?

Reply:
No that's fine if ESLint is okay with it!